### ISSUE TITLE
Node 14+ causes test to fail

### ISSUE THREAD

**Issue by dmcgill50** [appium/appium-mac2-driver#92]
Reverting to Node 12 allows tests to start passing again.

I believe the WARN for chokidar@2.1.8 is the reason for this failure.

```npm WARN deprecated request@2.88.0: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
```

**Comment by KazuCocoa** [comment]
What about using current `appium@rc`? It is 1.22 which will be next release.
The dependency has chokidar 3.5.1 https://github.com/appium/appium/blob/releases/1.22/npm-shrinkwrap.json#L2708

**Comment by dmcgill50** [comment]
I will update and get back to you on results.

**Comment by KazuCocoa** [comment]
no response for a while. Afaik, the reported dependency should be included in 1.22.
